packages/common-utils/src/__tests__/renderChartConfig.test.ts

```diff
@@ -104,28 +104,32 @@ describe('renderChartConfig', () => {
     const generatedSql = await renderChartConfig(config, mockMetadata);
     const actual = parameterizedQueryToSql(generatedSql);
     expect(actual).toBe(
-      'WITH RawSum AS (SELECT *,\n' +
-        '               any(Value) OVER (ROWS BETWEEN 1 PRECEDING AND 1 PRECEDING) AS PrevValue,\n' +
-        '               any(AttributesHash) OVER (ROWS BETWEEN 1 PRECEDING AND 1 PRECEDING) AS PrevAttributesHash,\n' +
-        '               IF(AggregationTemporality = 1,\n' +
-        '                  Value,IF(Value - PrevValue < 0 AND AttributesHash = PrevAttributesHash, Value,\n' +
-        '                      IF(AttributesHash != PrevAttributesHash, 0, Value - PrevValue))) as Rate\n' +
+      'WITH RawSum AS (SELECT toStartOfInterval(toDateTime(TimeUnix), INTERVAL 5 minute) AS `__hdx_time_bucket2`, AttributesHash, last_value(a.Value) AS `__hdx_value_high`,\n' +
```

Comment:
I suspect we don't need a unit test for this given the integration tests cover most of the use cases

Reply:
This is still light on cases but I see this suite getting built to check the edge case behaviors, e.g. value escaping, things outside assumptions, etc. The integration tests would have less cases but run against data in a clickhouse instance.

I did add a case that I would consider normally a unit test as an illustration of what I was thinking.